cmd/webhook/main.go

```diff
@@ -15,13 +15,16 @@ limitations under the License.
 package main
 
 import (
+	"github.com/aws/karpenter-core/pkg/apis"
 	"github.com/aws/karpenter-core/pkg/cloudprovider"
 	"github.com/aws/karpenter-core/pkg/webhooks"
+	"github.com/aws/karpenter/pkg/apis/awsnodetemplate/v1alpha1"
 	awscloudprovider "github.com/aws/karpenter/pkg/cloudprovider"
 	awscontext "github.com/aws/karpenter/pkg/context"
 )
 
 func main() {
+	apis.Resources[v1alpha1.SchemeGroupVersion.WithKind("AWSNodeTemplate")] = &v1alpha1.AWSNodeTemplate{}
```

Comment:
This feels like a strange, round-a-bout way to register the resources, is there a better registration flow that we could introduce?

Reply:
I want to overhaul the webhooks in a followon when I combine things. This is largely to unblock next week's release if there is one. I'm thinking about something like
```
operator.New().RegisterWebhooks(...).RegisterControllers(...) 
```
so you can do a builder pattern in both neutral and specific code.